README.md

```diff
@@ -1,3 +1,8 @@
+[![Platform:Windows](https://img.shields.io/badge/platform-Windows-lightgrey)](https://github.com/particl/particl-desktop/releases/latest)
+[![Platform:macOS](https://img.shields.io/badge/platform-macOS-lightgrey)](https://github.com/particl/particl-desktop/releases/latest)
+[![Platform:Linux](https://img.shields.io/badge/platform-Linux-lightgrey)](https://github.com/particl/particl-desktop/releases/latest)
+[![Twitter Follow](https://img.shields.io/twitter/follow/ParticlProject?label=follow%20us)](http://twitter.com/particlproject)
```

Comment:
Could this line be removed: it doesn't seem to belong here. It could possibly be moved further down, for example into the Community Chat section (but not necessarily in there... possibly in its own subsection?).

Reply:
Done.